.github/workflows/codecov-nightly.yml

```diff
@@ -0,0 +1,49 @@
+# Workflow derived from https://github.com/r-lib/actions/tree/v2/examples
+# Need help debugging build failures? Start at https://github.com/r-lib/actions#where-to-find-help
+name: Run Nightly Code Coverage
+# schedule from https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#schedule
+on:
+  schedule:
+    - cron: '0 0 * * *'
+jobs:
+  test-coverage:
+    runs-on: windows-latest
+    env:
+      GITHUB_PAT: ${{ secrets.GITHUB_TOKEN }}
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+      - name: Setup
+        uses: r-lib/actions/setup-r@v2
+        with:
+          use-public-rspm: true
+      - name: Setup Pandoc for word conversion
+        uses: r-lib/actions/setup-pandoc@v2
+      - name: Install ospsuite ecosystem and cran packages
+        run: |
+          Rscript -e "install.packages(c('covr', 'testthat', 'ggtext', 'knitr', 'rmarkdown', 'readr', 'remotes', 'dplyr', 'tidyr', 'purrr', 'patchwork', 'readxl','data.table'), repos='http://cran.rstudio.com/')"
+          Rscript -e "install.packages('https://github.com/Open-Systems-Pharmacology/rClr/releases/download/v0.9.1/rClr_0.9.1.zip', repos = NULL, type = 'binary')"
+          Rscript -e "download.file('https://ci.appveyor.com/api/projects/open-systems-pharmacology-ci/ospsuite-rutils/artifacts/ospsuite.utils.zip?pr=false', destfile = 'ospsuite.utils.zip', mode='wb'); install.packages('ospsuite.utils.zip', repos = NULL, type = 'binary')"
+          Rscript -e "download.file('https://ci.appveyor.com/api/projects/open-systems-pharmacology-ci/tlf-library/artifacts/tlf.zip?pr=false', destfile = 'tlf.zip', mode='wb'); install.packages('tlf.zip', repos = NULL, type = 'binary')"
+          Rscript -e "download.file('https://ci.appveyor.com/api/projects/open-systems-pharmacology-ci/ospsuite-r/artifacts/ospsuite.zip?pr=false', destfile = 'ospsuite.zip', mode='wb'); install.packages('ospsuite.zip', repos = NULL, type = 'binary')"
+      - name: Test coverage
+        run: |
+          covr::codecov(
+            quiet = FALSE,
+            clean = FALSE,
+            token = "${{ secrets.CODECOV_TOKEN }}",
```

Comment:
@Yuri05 
The tests are passing but I think we need to setup a [codecov token](https://docs.codecov.com/docs#step-2-get-the-repository-upload-token) to upload the coverage report

In my repository tests I got the following errors when 
- [token was not defined](https://github.com/pchelle/OSPSuite.ReportingEngine/actions/runs/5899514913/job/16002145888) 
- [token was ill defined](https://github.com/pchelle/OSPSuite.ReportingEngine/actions/runs/5899990137/job/16003409936)

Reply:
Is **this** the token?
https://github.com/Open-Systems-Pharmacology/OSPSuite.ReportingEngine/blob/f1768f80930a44d1c12885c4106471f3462f7844/appveyor.yml#L25-L26

Btw, the page you linked says: "Not required if your repo is using GitHub Actions, ..." Hmm.

Reply:
> Btw, the page you linked says: "Not required if your repo is using GitHub Actions, ..." Hmm.

I thought so too. But when running on my fork (which should be a public repo) I got the following error message: `Could not find a repository, try using repo upload token`

Then, I tried the token you showed and got `Invalid request parameters`. 
This is why was asking if we need to create a new one (or is it because the token may not work for my fork)